Typo in DataValidation examples (#1166)
exceljs/exceljs: README.md

  formulae: [15]
};

> // Specify Cell have to be a date before 1st Jan 2016

Reviewer: ```suggestion
// Specify Cell must be a date before 1st Jan 2016
```
